agents/temporal_difference.h

```diff
@@ -0,0 +1,28 @@
+#define LEARNING_RATE 0.02
+#define GAMMA 0.99
+#define MODEL_NAME "state_value.bin"
+
+#define CALC_STATE_NUM(x)                 \
+    {                                     \
+        x = 1;                            \
+        for (int i = 0; i < N_GRIDS; i++) \
+            x *= 3;                       \
+    }
+
+typedef struct td_agent {
+    char player;
+    float *state_value;
+} td_agent_t;
+
+int hash(char *table);
+char *hash_to_table(int hash);
```

Comment:
The function `hash_to_table` seems to be used internally, right? If so, don't expose it.